andela/ah-olympians-frontend
# 164047041 Users should be able to report an article

## Description ##
This pull request allows authenticated users to report articles that violate terms of agreement or are plagiarised to the Authors Haven's admin for removal and or any other actions as per the terms of engagement.

## How can This be tested ##
Clone the repo then:
- checkout to this branch
- do `npm install`
- do `npm install -g npm-install-peers`
- do `npm run coverage`
- do `npm start`
- go to `/login`
- then navigate to an article on the site. You should be able to view the Report button.

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## Related Pivotal Tracker stories ##
[#164047041](https://www.pivotaltracker.com/story/show/164047041)

## Reviewers ##
- @andela/olympian-devs
- @kelvinndmo 
- @winniekariuki
